packages/cli/src/models/compiler/Compiler.ts

```diff
@@ -0,0 +1,42 @@
+import { exec } from 'child_process';
+import { FileSystem, Contracts, Logger } from 'zos-lib';
+import Truffle from '../initializer/truffle/Truffle';
+import SolidityProjectCompiler from './solidity/SolidityProjectCompiler';
+import { CompilerOptions } from './solidity/SolidityContractsCompiler';
+
+const log = new Logger('Compiler');
+
+export default {
+  async call(): Promise<void> {
+    return Truffle.isTruffleProject()
```

Comment:
I guess the return statement is useless here

Reply:
Will remove on master.